projects/swimlane/ngx-ui/src/lib/components/dialog/large-format/large-format-dialog-content.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChild,
+  ElementRef,
+  EventEmitter,
+  HostBinding,
+  Input,
+  Output,
+  ViewEncapsulation
+} from '@angular/core';
+import { StepperComponent } from '../../stepper/stepper.component';
+import { TabsComponent } from '../../tabs/tabs.component';
+import { LargeFormatDialogFooterComponent } from './components/large-format-dialog-footer/large-format-dialog-footer.component';
+
+@Component({
+  selector: 'ngx-large-format-dialog-content',
+  exportAs: 'ngxLargeFormatDialogContent',
+  templateUrl: './large-format-dialog-content.component.html',
+  styleUrls: ['./large-format-dialog-content.component.scss', './large-format-dialog-content-drawer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None
+})
+export class LargeFormatDialogContentComponent {
+  // header-title inputs
+  @Input() title = '';
+  @Input() subtitle?: string;
+
+  // header-action inputs
+  @Input() actionTitle = 'Close';
+  @Input() dirty = false;
```

Comment:
Each use will have to implement it's own dirty check?

Reply:
I'm not sure how this LargeFormatDialog can implement its own dirty check since the content is projected from the consumers side?

Reply:
Talked to Mike. Will implement a unified Confirm to intercept the closeOrCancel click when `[dirty]` is true

Reply:
Working on unified Confirm language, waiting for Peggy to take a look.

Reply:
@nartc Let's use this:

![image](https://user-images.githubusercontent.com/1565092/113897011-b64fe300-9787-11eb-9f6f-1367ed60c675.png)

Reply:
Implemented